specs/specTemplate/sampleComponentSpec.md

```diff
@@ -109,14 +167,31 @@ Detailed description of how input behaves. Logic, corner cases, patterns, etc.
 
 
 
-### Touch
+#### **Touch**
 
 | Event | Action              | NOTE                    |
 | ----- | ------------------- | ----------------------- |
 | tap   | what happens on tap | Side note (if relevant) |
 | drag  |                     |                         |
 
 
+
+## RTL
```

Comment:
Should we have custom RTL handling per component?

Or just a general guideline?